InternshipProject#2/Models/Watcher.cs

```diff
@@ -11,6 +11,8 @@ public partial class Watcher
 
     public int? TicketId { get; set; }
 
+    public bool? IsDeleted { get; set; }
```

Comment:
As you added this new property, you have to make sure at reading/listing the watchers, only the watchers that have IsDeleted = false must be listed.